src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -66,11 +66,16 @@ public class ConvertFromMarkdownCommand : PSCmdlet
         /// </summary>
         protected override void BeginProcessing()
         {
-            _mdOption = this.CommandInfo.Module.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
-
-            if (_mdOption == null)
+            // If we have the moduleInfo then store are module scope variable
```

Comment:
I think we need comments that can better explain why we have different ways to handle this.

Reply:
Fixed.